### PR TITLE
Pytest sample stream

### PR DESCRIPTION
**Pull request by macikgozwa**
Summary:
 Adding a new test case to validate the `sampled streaming` changes on the telemetry service.
- The [PR](https://github.com/Azure/sonic-telemetry/pull/49) for the `sampled streaming` feature can be found here.
- The [PR](https://github.com/Azure/SONiC/pull/717) for documentation.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
A test case needed to cover a recently added telemetry service feature.

#### How did you do it?
The test case runs the mentioned query and checks the existence of expected keywords in the response.

#### How did you verify/test it?
Using Sonic KVM testbed.

#### Any platform specific information?
NA

#### Supported testbed topology if it's a new test case?
NA

### CODE REVIEW

File: tests/telemetry/test_telemetry.py [hunk 1]
```diff
@@ -138,6 +162,19 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, setup_streami
...
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
 
+def test_osbuild_version(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """ Test osbuild/version query.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    cmd = generate_client_cli(duthost=duthost, method=METHOD_GET, target="OTHERS", xpath="osversion/build")
+    logger.debug("Command to run: {0}".format(cmd))
+    show_gnmi_out = ptfhost.shell(cmd)['stdout']
+    logger.debug(show_gnmi_out)
```
Comment (wangxin) [inline tests/telemetry/test_telemetry.py:172]: This log is unnecessary. If log level set to DEBUG, the previous `ptfhost.shell(cmd)` call would log both invocation and result.

File: tests/telemetry/test_telemetry.py [hunk 2]
```diff
@@ -181,3 +218,19 @@ def test_sysuptime(duthosts, rand_one_dut_hostname, ptfhost, setup_streaming_tel
...
+def test_virtualdb_table_streaming(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """Run pyclient from ptfdocker to stream a virtual-db query multiple times.
+    """
+    logger.info('start virtual db sample streaming testing')
+
+    duthost = duthosts[rand_one_dut_hostname]
+    cmd = generate_client_cli(duthost=duthost, method=METHOD_SUBSCRIBE, update_count = 3)
+    logger.debug("Command to run: {0}".format(cmd))
+    show_gnmi_out = ptfhost.shell(cmd)['stdout']
+    logger.debug(show_gnmi_out)
```
Comment (wangxin) [inline tests/telemetry/test_telemetry.py:231]: Unnecessary log.

File: tests/telemetry/test_telemetry.py [hunk 3]
```diff
@@ -138,6 +162,19 @@ def test_telemetry_ouput(duthosts, rand_one_dut_hostname, ptfhost, setup_streami
     inerrors_match = re.search("SAI_PORT_STAT_IF_IN_ERRORS", result)
     pytest_assert(inerrors_match is not None, "SAI_PORT_STAT_IF_IN_ERRORS not found in gnmi_output")
 
+def test_osbuild_version(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """ Test osbuild/version query.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+    cmd = generate_client_cli(duthost=duthost, method=METHOD_GET, target="OTHERS", xpath="osversion/build")
+    logger.debug("Command to run: {0}".format(cmd))
```
Comment (wangxin) [inline tests/telemetry/test_telemetry.py:170]: This log is unnecessary. The `ptfhost.shell(cmd)` call would log it.

File: tests/telemetry/test_telemetry.py [hunk 4]
```diff
@@ -181,3 +218,19 @@ def test_sysuptime(duthosts, rand_one_dut_hostname, ptfhost, setup_streaming_tel
...
         pytest.fail("The value of system uptime was not updated correctly.")
+
+def test_virtualdb_table_streaming(duthosts, rand_one_dut_hostname, ptfhost, localhost):
+    """Run pyclient from ptfdocker to stream a virtual-db query multiple times.
+    """
+    logger.info('start virtual db sample streaming testing')
+
+    duthost = duthosts[rand_one_dut_hostname]
+    cmd = generate_client_cli(duthost=duthost, method=METHOD_SUBSCRIBE, update_count = 3)
+    logger.debug("Command to run: {0}".format(cmd))
```
Comment (wangxin) [inline tests/telemetry/test_telemetry.py:229]: Unnecessary log.

**Comment by wangxin** [comment]
retest vsimage please

**Comment by wangxin** [comment]
retest vsimage please

**Comment by macikgozwa** [comment]
retest vsimage please

**Review by wangxin — APPROVED** [review]

**Pull request merged**